server/index.js

```diff
@@ -10,12 +10,51 @@ app.listen(port, () => console.log(`App live on http://localhost:${port}`));
 app.use(express.static('public'));
 app.use(parser.json());
 app.use(parser.urlencoded({ extended: true }));
-app.get('/listings/:id', ({ params: { id } }, res) => {
-  Listing.findRandom({ id: { $ne: id } }).limit(12).exec((err, results) => {
-    if (err) {
-      throw err;
+
+app.get('/listings/:id', ({ params: { id } }, response) => {
+  Listing.findRandom({ id: { $ne: id } }).limit(12).exec((error, results) => {
```

Comment:
What is '$ne' here? Also, if you are finding by ID, why do you need a limit of 12? Shouldn't you only be returned one ID?

Reply:
My component grabs 12 random listings that aren't the current listing since I need to show 12 different listings. $ne is not equal